website/sidebars.js

```diff
@@ -341,5 +341,22 @@ module.exports = {
                 },
             ],
         },
+        {
+            label: 'Contributing',
+            type: 'category',
+            collapsed: false,
```

Comment:
I think this should be collapsed by default. No need to take up additional space in the side bar for something that's only applicable to a small number of users.

```suggestion
            collapsed: true,
```